src/NServiceBus.Testing.Fakes/OutgoingMessageExtensions.cs

```diff
@@ -16,7 +16,7 @@ public static IEnumerable<RepliedMessage<TMessage>> Containing<TMessage>(this IE
         {
             return repliedMessages
                 .Where(x => x.Message is TMessage)
-                .Select(x => new RepliedMessage<TMessage>((TMessage)x.Message, x.Options));
+                .Select(x => new RepliedMessage<TMessage>((TMessage) x.Message, x.Options));
```

Comment:
I have to say I really hate this style convention of having a space here. Would anyone be opposed to changing this?